image/image.go

```diff
@@ -143,17 +158,23 @@ func CreateRuntimeBundleLayout(src, dest, ref, root string) error {
 	return createRuntimeBundle(newPathWalker(src), dest, ref, root)
 }
 
-// CreateRuntimeBundle walks through the given .tar file and
-// creates an OCI runtime bundle in the given destination dest
-// or returns an error if the unpacking failed.
-func CreateRuntimeBundle(tarFile, dest, ref, root string) error {
+// CreateRuntimeBundleFile opens the file pointed by tarFile and calls
+// CreateRuntimeBundle.
```

Comment:
s/CreateRuntimeBundle/createRuntimeBundle ?

And, CreateRuntimeBundle which seems to be unused should be removed?